### ISSUE TITLE
Validation of the incoming messages

### ISSUE THREAD

**Issue by callmekatootie** [topcoder-platform/vorbote#10]
Provide validation of incoming messages. Perhaps allow the user to activate this when creating a webhook?

**Comment by cwdcwd** [comment]
Yes, this would be great to make an optional feature with a flag at setup/update time for whether or not it performs the validation

**Comment by cwdcwd** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Validation of the incoming messages](https://trello.com/c/2DOtmw5Z/11-validation-of-the-incoming-messages)

**Comment by callmekatootie** [comment]
Need to think this through - not sure what I had in mind when I created this ticket... We already have filters (rules) so I am not sure what would "validation" feature be in this context...
